src/concrete/ml/torch/hybrid_model.py

```diff
@@ -519,12 +508,17 @@ def _save_fhe_circuit(self, path: Path, via_mlir=False):
 
         model_path = Path(path)
         for module_name in self.module_names:
+            onnx_model = self.private_q_modules[  # pylint: disable=protected-access
```

Comment:
there is not need to access the private `_onnx_model`, `QuantizedModule` instances have a public property for them , you can directly call `onnx_model` normally

Reply:
Ah yes!

Reply:
Fixed